otx/api/entities/image.py

```diff
@@ -5,7 +5,7 @@
 #
```

Comment:
This looks like to take care more general representation of image entity as we did in Datumaro media refactoring. Is this mandatory to consume Apache Arrow formatted data through Datumaro?

Reply:
Reverted in https://github.com/openvinotoolkit/training_extensions/pull/2009/commits/f993b6607011b4d6740ccf971e3408a8f3a8de6d